pkg/cmd/service/service_test.go

```diff
@@ -93,6 +93,11 @@ func ensureServiceUp(t *testing.T, transportCredentials credentials.TransportCre
 		return nil
 	}, policy)
 	require.NoError(t, err)
+
+	if useHTTPToo {
+		_, err = retryablehttp.NewClient().Get("http://localhost:8080/healthz")
```

Comment:
`openFGAServerAddr` has this config rather than hardcoding. That way when this gets refactored we don't have to look it up.

Might be worth also putting it in the `backoff` function after the GRPC healthcheck.

Reply:
Wait, why do we have the HTTP option instead of just using the single RPC health check? I feel like this is extra code that isn't needed.

Reply:
The RPC gateway might boot up while the HTTP listener might not. We want to ensure both are up.

Reply:
What Yamil said. I suppose this would be fixed with retryable client everywhere.

Another problem however, was some tests were failing because we could cancel the context when the HTTP server was in the middle of starting up resulting in an error. This is probably fixed by disabling the HTTP server in such tests.

In any case, I think this adds robustness.